### PR TITLE
Create Post “kodi-21-0-omega-release/index”

### PR DESCRIPTION
**Pull request by ProfYaffle**
Automatically generated by Decap CMS

### CODE REVIEW

File: src/content/article/kodi-21-0-omega-release/index.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+With this release we have seen over 3,750 commits since v20.0 "Nexus" was available to the world on January 15th, 2023. As always, this has been a huge effort from many contributors, testers and supporters. Thanks go out to our team members, plus everyone in our community who sent in a pull request, tested and gave feedback, or provided support to users on the forum. We always appreciate your contribution to making Kodi better.
+
+Let's review some of the changes you can expect:
+
+### Major Features
+
+**FFmpeg 6**
+
+Kodi relies significantly on FFmpeg to do a lot of heavy lifting for us. Many developers have contributed to upgrading the project to make use the the newer FFmpeg releases over the past 15 months, initially starting with targeting FFmpeg 5, but later updating to FFmpeg 6. We are always greatly appreciative of the work of all of the Open Source software projects that we rely on on our journey to bring you Kodi.
```
Comment (JohnVeness) [inline src/content/article/kodi-21-0-omega-release/index.md:23]:
```suggestion
Kodi relies significantly on FFmpeg to do a lot of heavy lifting for us. Many developers have contributed to upgrading the project to make use of the newer FFmpeg releases over the past 15 months, initially starting with targeting FFmpeg 5, but later updating to FFmpeg 6. We are always greatly appreciative of the work of all of the Open Source software projects that we rely on on our journey to bring you Kodi.
```
Also, might want to do something to remove the two "on"s in a row, e.g. simplify to "that we rely on to bring you Kodi."

File: src/content/article/kodi-21-0-omega-release/index.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+**DolbyVision On-the-Fly Profile Conversion**
+
+For our Android users, a fantastic contribution from **quietvoid** allows users to convert some less-well supported DV profile types to more well-supported profiles.
```
Comment (JohnVeness) [inline src/content/article/kodi-21-0-omega-release/index.md:27]:
```suggestion
For our Android users, a fantastic contribution from **quietvoid** allows users to convert some less well-supported DV profile types to more well-supported profiles.
```

File: src/content/article/kodi-21-0-omega-release/index.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+**Availability**
+
+Kodi 21.0 "Omega" is being pushed to the usual channels right now, both our own [download](https://kodi.tv/download) pages and various app stores/repositories, so should be with your devices shortly. It's worth noting that, for the first time in a while, we've cleared all known blockers, so all supported platforms will be updated just as soon as the various app stores process things.
+
+If you are an android user and wish to update to Kodi 21.0 asap, you can either sideload, or join the [Google Play - Kodi testing track](https://play.google.com/apps/testing/org.xbmc.kodi) which will get the release submitted asap. The Open release will be promoted from the testing track in a week or two. We strongly advise Android users who use alternate skins to follow up with their skin developer, or to disable auto update before this time period.
```
Comment (JohnVeness) [inline src/content/article/kodi-21-0-omega-release/index.md]:
```suggestion
If you are an Android user and wish to update to Kodi 21.0 ASAP, you can either sideload, or join the [Google Play - Kodi testing track](https://play.google.com/apps/testing/org.xbmc.kodi) which will get the release submitted very soon. The Open release will be promoted from the testing track in a week or two. We strongly advise Android users who use alternate skins to follow up with their skin developer, or to disable auto update before this time period.
```

**Comment by JohnVeness** [comment]
Darn, missed the merge. Shall I do a PR?

**Comment by ProfYaffle** [comment]
Yes, sorry - I did it in one go to get it out quickly. Your changes should be merged now, though, so thank you for the second pair of eyes.

**Comment by JohnVeness** [comment]
Thanks :)

**Pull request merged**